src/Writer/FlushableWriter.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Ddeboer\DataImport\Writer;
+
+interface FlushableWriter
```

Comment:
But flush should also be called at the end of the workflow. And there is quite a big risk of misuse, like it appears in the code as well (see my earlier comment bellow)